controllers/component_build_controller_pac.go

```diff
@@ -888,11 +888,24 @@ func generatePACRepository(component appstudiov1alpha1.Component, config map[str
 func (r *ComponentBuildReconciler) generatePaCPipelineRunConfigs(ctx context.Context, component *appstudiov1alpha1.Component, gitClient gp.GitProviderClient, pacTargetBranch string) ([]byte, []byte, error) {
 	log := ctrllog.FromContext(ctx)
 
-	pipelineRef, additionalPipelineParams, err := r.GetPipelineForComponent(ctx, component)
-	if err != nil {
-		return nil, nil, err
+	var pipelineName string
+	var pipelineBundle string
+	var additionalPipelineParams []tektonapi.Param
+	var pipelineRef *tektonapi.PipelineRef
+	var err error
+
+	pipelineLabelUsed := true
```

Comment:
I see no point in this variable. The value can be easily obtained from the component.

Reply:
removed